missile.pde

```diff
@@ -23,15 +23,15 @@ class Missile {
   float fuel;
   boolean live;
   int ttl;
-  int lifeSpan = 600;
+  int lifeSpan = 500;
   float rot; // where the missile is facing
   Ship parent;
   Ship enemyShip;
   float enemyRange = windowSize; // can track you over entire screen right now
   float enemyAngleTolerance = 360;
   float mass = .4;
   float missileLaunchForce = 1;
-  float missileSmartFactor = 0.046;
+  float missileSmartFactor = 0.045;
```

Comment:
why did you make the missile smarter, dude?

Reply:
i made it less smart